src/views/Activate/ActivatePassport.js

```diff
@@ -44,6 +50,21 @@ export default function ActivatePassport() {
   const leftHalf = useBreakpoints([false, false, 1]);
   const rightHalf = useBreakpoints([false, false, 2]);
 
+  // addresses are not derived until we set[Urbit]Wallet(), so do that inline
```

Comment:
Pretty sure you can just say `wallet.ownership.keys.address` though, since it's an Urbit Wallet?